gatsby-node.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Did this not work when built as a plugin or did you extract it for other reasons?

Reply:
This is the proper way to handle modifying the webpack stuff in Docz 2 I believe. Im open to moving it back in, but I do like the idea of keeping the `.doczrc` a little bit cleaner.